### DISCUSSION TITLE
rhythmic division

### DISCUSSION THREAD

**Discussion by Bobbyjoe74** [amiika/ziffers#63] [Q&A] [answered]
Hello Amiika and everyone !
I'm into live coding for over a year now and I just started to investigate the wonderful Ziffers.
I have a question about rhythm divisions : In regular music notation, a "quarter note" is equal to a beat (so for example if the BPM is 60, then you'll have a quarter note every second) ; a half note is twice longer (so equals to two beats) and a whole note 4 times longer (so a whole note equals 4 beats)...etc.
But I noticed that in Ziffers it's the whole note that correspond to a beat (so it's basically 4 times faster than it should be).
Is there a reason for that ? 
As a musician, it is super confusing to use as it is now.

Thanks a lot !

Best.

**Comment by amiika** [comment]
Hi Bobbyjoe!

Thanks for asking this question. As a hobbyist I'v just defined it as it is now. Kind of been inventing my own rules I guess.

But you are right and as of now whole note corresponds to a beat (same as sleep 1.0 in Sonic Pi) and quarter note is now 0.25 of the beat.

I'll fix this, but I suppose for now to get it right use:

```
use_bpm 60/4

z1 "q1"
```

**Comment by amiika** [comment, marked as answer]
Pushed a fix for the BPM. Let me know if it works for you now :)
